README.md

```diff
@@ -981,14 +981,14 @@ module to capture frames of audio and automatically pass it to the wake word eng
 
 [Porcupine](/binding/react-native/src/porcupine.tsx) provides low-level access to the wake word engine for those
 who want to incorporate wake word detection into a already existing audio processing pipeline. `Porcupine` also has
-`fromKeywords` and `fromKeywordPaths` static constructors.
+`fromBuiltInKeywords` and `fromKeywordPaths` static constructors.
 
 ```javascript
 const accessKey = "${ACCESS_KEY}" // AccessKey obtained from Picovoice Console (https://picovoice.ai/console/)
 
 async createPorcupine(){
     try{
-        this._porcupine = await Porcupine.fromKeywords(accessKey, ["picovoice"]);
+        this._porcupine = await Porcupine.fromBuiltInKeywords(accessKey, ["picovoice"]);
```

Comment:
update to BuiltInKeywords.PICOVOICE (and double-check the readmes for any more of these)